src/test/java/net/openhft/chronicle/queue/impl/single/NotCompleteTest.java

```diff
@@ -170,9 +171,10 @@ private String cleanQueueDump(String from) {
     }
 
     private void doWrite(ChronicleQueue queue, BiConsumer<PersonListener, ChronicleQueue> action) {
-        ExcerptAppender appender = queue.acquireAppender();
-        PersonListener proxy = appender.methodWriterBuilder(PersonListener.class).get();
-        action.accept(proxy, queue);
+        try (ExcerptAppender appender =queue.createAppender()) {
+            PersonListener proxy = appender.methodWriterBuilder(PersonListener.class).get();
+            action.accept(proxy, queue);
+        }
```

Comment:
I think you can use `ChronicleQueue#methodWriter(PersonListener.class)` here and avoid (or at least hide) the use of `xxxAppender` altogether.

Reply:
Nice! That's changed